src/apiCall.js

```diff
@@ -0,0 +1,63 @@
+const url = 'https://devlaps-backend.herokuapp.com';
```

Comment:
What do you think about storing URL's in environment variables such that we can use the appropriate development/production URLs depending on which environment we're running the app?